tests/geomodel/core/test-get-dependent-entities.cpp

```diff
@@ -111,9 +113,9 @@ void test_template( GeoModel3D& geomodel,
 void test_on_top_region( GeoModel3D& geomodel )
 {
     // Solution:
-    // Corners: 31, 33, 54, 55, 56, 57, 58, 93, 118, 128, 129.
-    // Lines: 41, 43, 68, 69, 70, 71, 72, 73, 131, 135, 144, 177, 182, 203, 205,
-    // 207, 210, 233, 234, 238.
+    // Corners: 31, 33, 54, 55, 56, 57, 58, 99, 125, 138, 139.
+    // Lines: 41, 43, 68, 69, 70, 71, 72, 73, 137, 141, 151, 184, 189, 213, 215,
```

Comment:
What do you think about removing these lines? This is a dupplicate information. No need to maintain both.